catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/parser/ParserTest.java

```diff
@@ -152,7 +129,7 @@ public void testEmptyInput() {
 
 	@Test
 	public void testFuctionalAndSimpleBracketsCorrection() {
-		List<InternToken> internTokenList = new LinkedList<InternToken>();
+		List<InternToken> internTokenList = new LinkedList<>();
 
 		internTokenList.add(new InternToken(InternTokenType.FUNCTION_NAME, Functions.ABS.name()));
 		internTokenList.add(new InternToken(InternTokenType.FUNCTION_PARAMETERS_BRACKET_OPEN));
```

Comment:
Future Refactoring Ticket (Training). Next step could be to rewrite this class to a parameterized test.

Reply:
Is there already a ticket?